tests/gcp_iam_roles/dynatrace-gcp-function-e2e-tests-role.yaml

```diff
@@ -0,0 +1,28 @@
+title: E2E tests dynatrace-gcp-function
+description: Role for Dynatrace GCP function E2E tests
+stage: GA
+includedPermissions:
+  - cloudfunctions.functions.call
+  - cloudfunctions.functions.create
+  - cloudfunctions.functions.delete
+  - cloudfunctions.functions.get
+  - cloudfunctions.functions.invoke
+  - cloudfunctions.functions.list
+  - cloudfunctions.functions.sourceCodeGet
+  - cloudfunctions.functions.sourceCodeSet
+  - cloudfunctions.functions.update
+  - cloudfunctions.locations.get
+  - cloudfunctions.locations.list
+  - cloudfunctions.operations.get
+  - cloudfunctions.operations.list
+  - iam.roles.create
```

Comment:
iam permissions are needed for helm deployment - add them also to deployment role